WordPress/src/main/java/org/wordpress/android/ui/prefs/ProfileInputDialogFragment.java

```diff
@@ -0,0 +1,101 @@
+package org.wordpress.android.ui.prefs;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import org.wordpress.android.R;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.widgets.OpenSansEditText;
+import org.wordpress.android.widgets.WPTextView;
+
+public class ProfileInputDialogFragment extends DialogFragment {
+
+    private static final String TITLE_TAG = "title";
+    private static final String INITIAL_TEXT_TAG = "initial_text";
+    private static final String HINT_TAG = "hint";
+    private static final String IS_MULTILINE_TAG = "is_multiline";
+    private static final String TEXT_VIEW_ID_TAG = "text_view_id";
+
+    public static ProfileInputDialogFragment newInstance(String title,
+                                   String initialText,
+                                   String hint,
+                                   boolean isMultiline,
+                                   int textViewId) {
```

Comment:
Hey @rishabh7m , sorry for the delayed response!

So, yeah, an `int` parameter is just fine here.

But, even though this might seem like nitpicking, I'd suggest to name it after a more generic name than "textViewId". For ProfileInputDialogFragment, this integer is nothing more than an identifier to pass back to the callback. In that respect, it might be a TextView id or it might be any other identifier basically. I would suggest naming it `callbackId` or something similarly generic. Cheers!